packages/gatsby-cli/src/reporter/loggers/ink/components/messages.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import { Box, Color, ColorProps } from "ink"
+
+import { ActivityLogLevels, LogLevels } from "../../../constants"
+
+const ColorSwitcher: React.FC<ColorProps> = ({ children, ...props }) => (
```

Comment:
```suggestion
const ColorSwitcher: FunctionComponent<ColorProps> = ({ children, ...props }) => (
```

Reply:
I think these are exactly the same because its definition is 
```ts
type FC<P = {}> = FunctionComponent<P>;
```
I prefer `React.FC` to `React. FunctionComponent` because of its shortness and simplicity
What do you think?

Reply:
I always use `FunctionComponent` so it's a lot clearer what it means